obf_creation/rendering_types.xml

```diff
@@ -1170,11 +1170,21 @@
 		<type tag="start_ele" additional="text" minzoom="3"/>
 		<type tag="ele_graph" additional="text" minzoom="3"/>
 
-		<type tag="width" additional="true" minzoom="3"/>
```

Comment:
Why remove **width**?

Reply:
https://github.com/osmandapp/OsmAnd-resources/pull/1139/commits/6528f37b261e0e267bdb4b08340e4ba0b210cd1d

It's used to display width=thin|medium|bold of user gpx tracks.
Now replaced with gpx_width as well as most other gpx_ attributes.

Reply:
resolved with xmd5a